### DISCUSSION TITLE
Is there any plan or timeline to upgrade to angular 16?

### DISCUSSION THREAD

**Discussion by frank1forward** [ncstate-sat/popover#279] [Q&A] [answered]
Whats the process to upgrade

**Comment by wjaspers** [comment]
There are two forks whom both have upgraded to Angular 16.

 See this issue:
https://github.com/ncstate-sat/popover/issues/270

Both forks offer different version numbers.
@igor2017/sat-popover
@wjaspers/sat-popover

**Comment by isaackehle** [comment, marked as answer]
Upgrades happened today. Enjoy!

#287 #286 #285 #284 #283
